build.sbt

```diff
@@ -31,6 +31,16 @@ def http4sDependencies(scalaVersion: String): Seq[sbt.ModuleID] =
     Seq("org.http4s" %% "http4s-core" % "0.21.0-M5",
         "org.http4s" %% "http4s-dsl" % "0.21.0-M5" % "test")
 
+lazy val monixLogbackCore : Project = (project in file("monix-logback-core"))
```

Comment:
maybe just `monix-logback`? Is there anything that's "core" about it? (if we had a zio correlator, that wouldn't depend on this module, for example?)